openfisca_core/entities.py

```diff
@@ -1,5 +1,7 @@
 # -*- coding: utf-8 -*-
 
+import traceback
+
```

Comment:
I may have mis-read the code, I thought there was a line break between two standard library imports 👌.
Yep, we should follow these rules.